packages/gasket-engine/test/exec.test.js

```diff
@@ -10,6 +14,34 @@ describe('The exec method', () => {
     hooks: {
       eventA() {
         return Promise.resolve(1);
+      },
+      eventB: {
+        timing: {
+          after: ['pluginB']
+        },
+        handler: async function () {
```

Comment:
```suggestion
        async handler() {
```
Should we consider employing a named function in this context as a best practice to potentially enhance debugging by providing more informative stack traces?